admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,253 @@
+determine how many instances of an input pod can be scheduled on the cluster
+before it is exhausted of resources in order to provide a more accurate
+estimation.
+
+[NOTE]
+====
+The remaining allocatable capacity is a rough estimation, because it does not
+count all of the resources being distributed among nodes. It analyzes only the
+remaining resources and estimates the available capacity that is still
+consumable in terms of a number of instances of a pod with given requirements
+that can be scheduled in a cluster.
+
+Also, pods might only have scheduling support on particular sets of nodes based
+on its selection and affinity criteria. As a result, the estimation of which
+remaining pods a cluster can schedule can be difficult.
+====
+
+The cluster capacity analysis tool can be run either as a stand-alone utility
+from the command line, or in a pod inside an {product-title} cluster.
+
+[[cluster-capacity-running-analysis]]
+== Running Cluster Capacity Analysis on the Command Line
+
+To run the tool on the command line:
+
+----
+$ cluster-capacity --kubeconfig <path-to-kubeconfig> \
+    --podspec <path-to-pod-spec>
+----
+
+The `--kubeconfig` option indicates your Kubernetes configuration file,
+and the `--podspec` option indicates a sample pod specification file, which
+the tool uses for estimation. The `podspec` specifies its resource requirements
+as `limits` or `requests`. The cluster capacity tool takes the pod's resource
+requirements into account for its estimation analysis.
+
+An example of the pod specification input is:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: small-pod
+  labels:
+    app: guestbook
+    tier: frontend
+spec:
+  containers:
+  - name: php-redis
+    image: gcr.io/google-samples/gb-frontend:v4
+    imagePullPolicy: Always
+    resources:
+      limits:
+        cpu: 150m
+        memory: 100Mi
+      requests:
+        cpu: 150m
+        memory: 100Mi
+----
+
+The `--verbose` option can also be provided as part
+of the above command to output a detailed description of how many pods can be
+scheduled on each node in the cluster:
+
+----
+$ cluster-capacity --kubeconfig <path-to-kubeconfig> \
+    --podspec <path-to-pod-spec> --verbose
+----
+
+The output will look similar to the following:
+
+----
+small-pod pod requirements:
+	- CPU: 150m
+	- Memory: 100Mi
+
+The cluster can schedule 52 instance(s) of the pod small-pod. <1>
+
+Termination reason: Unschedulable: No nodes are available that match all of the
+following predicates:: Insufficient cpu (2).
+
+Pod distribution among nodes:
+small-pod
+	- 192.168.124.214: 26 instance(s)
+	- 192.168.124.120: 26 instance(s)
+----
+<1> The number of estimated pods that can be scheduled on the cluster. 
+
+[[admin-guide-running-cluster-capacity-inside-pod]]
+== Running Cluster Capacity as a Job Inside of a Pod
+
+Running the cluster capacity tool as a job involves using a `ConfigMap`.
```

Comment:
@derekwaynecarr @aveshagarwal What are the advantages/disadvantages of doing it in a job inside a pod, over on the command line? Anything I can add to give the reader more info?